frontend/src/components/map/MapComponent.tsx

```
@@ -1,101 +1,86 @@
+import { MapContainer, TileLayer, useMap, Marker, Polygon } from "react-leaflet"
+import L from "leaflet"
+import "leaflet/dist/leaflet.css"
+import type { Location } from "@/lib/types"
+import { NavigationControls } from "./NavigationControls"
+import { Button } from "@/ui/button"
+import { MapIcon } from "lucide-react"
+import { Popover, PopoverContent, PopoverTrigger } from "@/ui/popover"
 
 // Fix for default markers
-delete (L.Icon.Default.prototype as any)._getIconUrl;
+delete (L.Icon.Default.prototype as any)._getIconUrl
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)